snomed/com.b2international.snowowl.snomed.datastore/src/com/b2international/snowowl/snomed/datastore/request/SnomedConceptCreateRequestBuilder.java

```diff
@@ -34,7 +33,6 @@
  */
 public final class SnomedConceptCreateRequestBuilder extends SnomedComponentCreateRequestBuilder<SnomedConceptCreateRequestBuilder> {
 
-	private DefinitionStatus definitionStatus = DefinitionStatus.PRIMITIVE;
```

Comment:
Keep the `definitionStatus` property on this request type. (to be backward compatible and support hybrid `OWL Axiom + old` stated mode)